docs/MyFirstPullRequest.md

```diff
@@ -0,0 +1,48 @@
+# My First Pull Request
+
+Okay. You are a member of the Rocket 2 team. You have successfully set up your
+local development environment. You have found an issue that you want to work
+on. Now, what to do?
+
+## Setting up branches
+
+Before you make any changes, you should first set up your own branch. It is
+common convention to name your branch:
+
+```
+<username>/#<issue-number>-<description-of-fix>
+```
+
+So if your issue is [#153 Read from configuration][#153], you would name it
+`rwblickhan/#153-read-from-config`. The name needs to be concise, descriptive,
+and, well, have your name and number, so to speak.
+
+## Before-Pull-Request checklist
+
+- All unit tests pass (`scripts/build_check.sh`)
```

Comment:
It might be good to note that `build_check.sh` (at least as far as I remember, unless you've changed something) also runs listing etc, not just unit tests.